spec/features/user_enters_valid_email_spec.rb

```diff
@@ -1,5 +1,39 @@
 require 'rails_helper'
 
-RSpec.feature 'Sign-up email', type: :feature do
-  
+RSpec.feature 'Sign-in email', type: :feature do
+  scenario "user enter blank password with valid e-mail" do
+    sign_up_with_sign_out
+    fill_in('Email', :with => 'user@gmail.com')
+    click_button "Log in"
+    expect(page).to have_content("Invalid Email or password.")
+  end
+
+  scenario "user enter blank password and blank e-mail" do
+    sign_up_with_sign_out
+    click_button "Log in"
+    expect(page).to have_content("Invalid Email or password.")
```

Comment:
"Invalid Email or password" did you create this error message?

Reply:
no, it was create by devise i think but where never found where specifically its is.